lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -766,6 +775,11 @@ def process(
             If random_ordering is True, the post-processed forecasts are
             reordered randomly, rather than using the ordering of the
             raw ensemble.
+        fixed_random_seed : Integer or None
+            If fixed_random_seed is an integer, the integer value is used for
+            the random seed.
+            If fixed_random_seed is None, no random seed is set, so the random
+            values generated are not predictable.
```

Comment:
One could argue that a good RNG does not produce "predictable" output, even with a fixed initial seed, it's simply that it returns the same sequence with a fixed initial seed. :)

...I'm just nitpicking, but I would say the sequence itself is reproducible, but its values are not predictable.